src/main/webapp/js/controllers.js

```diff
@@ -93,8 +98,41 @@ angular
                 'techDetailsController',
                 function($scope, $http, $location, $routeParams, $timeout,
                         $rootScope) {
-                    var id = getParameterByName('id');
-                    alert(id);
+
+                    $timeout(function() {
+                        getTechnollogy();
+                    }, 200);
+
+                    function getTechnollogy() {
+                        var host = location.host;
+                        var complement = '/_ah/api/';
+                        var rootUrl = 'http://' + host + complement;
+                        gapi.client.load('rest', 'v1', callBackLoaded, rootUrl);
+                        // mockTechnology();
+                    };
+
+                    function mockTechnology() {
+                        $scope.name = "Nome da tecnologia de id ";
+                        $scope.description = "Suco de cevadiss, é um leite divinis, qui tem lupuliz, matis, aguis e fermentis. Interagi no mé, cursus quis, vehicula ac nisi. Aenean vel dui dui. Nullam leo erat, aliquet quis tempus a, posuere ut mi. Ut scelerisque neque et turpis posuere pulvinar pellentesque nibh ullamcorper. Pharetra in mattis molestie, volutpat elementum justo. Aenean ut ante turpis. Pellentesque laoreet mé vel lectus scelerisque interdum cursus velit auctor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ac mauris lectus, non scelerisque augue. Aenean justo massa.";
+                        $scope.citDesc = "CI&T description";
+                        $scope.image = "https://storage.googleapis.com/tech-gallery-assets/imagesLogo/adf.png";
+                    };
+
+                    function callBackLoaded() {
+                        var idTechnology = getParameterByName('id');
+                        var req = {
+                            id : idTechnology
+                        };
+                        gapi.client.rest.getTechnology(req).execute(
+                                function(data) {
+                                    $scope.name = data.name;
+                                    $scope.description = data.description;
+                                    $scope.citDesc = data.citDesc;
+                                    $scope.image = data.image;
```

Comment:
Please avoid mentions to CI&T on the source-code. This application should be company agnostic as much as possible.